website/content/en/docs/upgrading/v1-migration.md

```diff
@@ -229,9 +229,11 @@ for [EKS Pod Identity ABAC policies](https://docs.aws.amazon.com/eks/latest/user
    ```
```

Comment:
Can we make these changes to the `preview` directory? These are the changes that actually get carried forward from version to version